suit_generator/suit/manifest.py

```diff
@@ -205,6 +205,11 @@ def from_obj(cls, obj: dict) -> SuitUint:
 
             binary_data = SuitEnvelopeTagged.return_processed_binary_data(obj["envelope"])
             return super().from_obj(len(binary_data))
+        elif "file_direct" in obj.keys():
```

Comment:
Comment - not issue: `file_direct` is allright. As an alternative, `file_value` might be used.